.circleci/config.yml

```diff
@@ -44,6 +59,9 @@ commands:
       # Setup the database
       - run: bundle exec rake db:setup
 
+      # Build assets
+      - run: yarn run build-dev:js && yarn run build-dev:css
+
```

Comment:
@pablobm I like the `:js`, but I also think it is okay to leave it out because it seems to be a pattern that `build` builds js by default.

For example, when we generate a new rails app with some javascript and css setup:

```
cd /tmp/
rails new testing --javascript=esbuild --css=sass

cd testing
cat package.json
```

We get

```js
  // ...
  "scripts": {
    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=/assets",
    "build:css": "sass ./app/assets/stylesheets/application.sass.scss:./app/assets/builds/application.css --no-source-map --load-path=node_modules"
  }
```

Reply:
Oh right. Good point, let's stick to convention then :+1: